test/kv-secret-engine.yaml

```diff
@@ -5,6 +5,6 @@ metadata:
 spec:
   authentication: 
     path: kubernetes
-    role: kv-engine-admin
+    role: policy-admin
```

Comment:
can you revert this change, it should not be needed

Reply:
change has been reverted.